applications/serial_lte_modem/doc/slm_data_mode.rst

```diff
@@ -52,15 +56,23 @@ The SLM application sends an *OK* response when it successfully enters data mode
 Sending data in data mode
 =========================
 
+Any arbitrary data received from the MCU is sent to LTE network *as-is*.
+
 If the current sending function succeeds and :ref:`CONFIG_SLM_DATAMODE_URC <CONFIG_SLM_DATAMODE_URC>` is defined, the SLM application reports back the total size as ``#XDATAMODE: <size>``.
 The ``<size>`` value is a positive integer.
+This URC can also be used to impose flow control on uplink sending.
```

Comment:
What is URC? A quick google seems to get stuck on a rugby team.

Reply:
Unsolicited Result Code, a term used since old-time Hayes modem
https://embeddedfreak.wordpress.com/2008/08/19/handling-urc-unsolicited-result-code-in-hayes-at-command/

Reply:
```suggestion
This Unsolicited Result Code (URC) can also be used to impose flow control on uplink sending.
```

Reply:
URC is a common terminology that could be used elsewhere, too. If we really need to give the full name, we should describe it in the description doc. Quote a competitor's AT manual:

``1.7. Unsolicited Result Code
As an Unsolicited Result Code and a report message, URC is not issued as part of the response related
to an executed AT command. URC is issued by BG95/BG77/BG600L-M3 without being requested by the
TE and it is issued automatically when a certain event occurs. Typical events leading to URCs are
incoming calls (RING), received short messages, high/low voltage alarm, high/low temperature alarm, etc.``

Reply:
I'll update accordingly.